iron-flex-layout.html

```diff
@@ -10,6 +10,23 @@
 
 <link rel="import" href="../polymer/polymer.html">
 
+<script>
+/**
+The `<iron-flex-layout>` component provides simple ways to use [CSS flexible box layout](https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Flexible_boxes), also known as flexbox. This component provides two different ways to use flexbox:
+
+1. [Layout classes](https://github.com/PolymerElements/iron-flex-layout/tree/master/classes). The layout class stylesheet provides a simple set of class-based flexbox rules. Layout classes let you specify layout properties directly in markup.
+
+2. [Custom CSS mixins](https://github.com/PolymerElements/iron-flex-layout/blob/master/iron-flex-layout.html). The mixin stylesheet includes custom CSS mixins that can be applied inside a CSS rule using the `@apply` function.
+
+A complete [guide](https://elements.polymer-project.org/guides/flex-layout) to `<iron-flex-layout>` is available.
+
+@demo demo/index.html
+**/
+Polymer({
+  is:'iron-flex-layout'
```

Comment:
What do you think about logging a warning incase someone creates an iron-flex-layout element. Since it is not intended to be used this was and this is more of a compliance thing for our tooling ?